package.json

```diff
@@ -119,6 +144,7 @@
         "visible": true,
         "subPermissions": [
           "settings.finance.enabled",
+          "ui-finance.settings.view",
```

Comment:
I think you can remove the below permissions in the list of sub-permissions when you add "ui-finance.settings.view" here, as they will be inherited.

"settings.finance.enabled",
          "finance.expense-classes.collection.get",
          "finance.fiscal-years.collection.get",
          "finance.fund-codes-expense-classes.collection.get",
          "finance.fund-types.collection.get",
          "users.collection.get"